posthog/api/team.py

```diff
@@ -138,15 +138,16 @@ def create(self, validated_data: Dict[str, Any], **kwargs) -> Team:
         serializers.raise_errors_on_nested_writes("create", self, validated_data)
         request = self.context["request"]
         organization = self.context["view"].organization  # Use the org we used to validate permissions
-        if validated_data.get("is_demo", False):
-            matrix = HedgeboxMatrix(n_clusters=settings.DEMO_MATRIX_N_CLUSTERS)
-            manager = MatrixManager(matrix, use_pre_save=True)
         with transaction.atomic():
             team = Team.objects.create_with_data(**validated_data, organization=organization)
             request.user.current_team = team
             request.user.save()
-            if "manager" in locals():
```

Comment:
I missed this in the original PR, but `locals()` shouldn't really be ever used. It works but is super hacky and harder to read.

Reply:
Ok sounds good! My python skills are still in the works 😄